### ISSUE TITLE
Okta question

### ISSUE THREAD

**Issue by oferRounds** [openid/AppAuth-Android#484]
I’m doing my first steps with Okta, and find it a bit hard to follow their Docs, and their support is not that responsive, so if that‘s ok, I‘ll try to post here my 2 questions:

1. The ogranization I work for already has a SAML Okta app for our web-desktop login. Can I set the same app to support also open-id/mobile native login?

2. It says here:
> As with any Okta application, make sure you assign Users or Groups to the OpenID Connect Client. Otherwise, no one can use it. 

The question: how can I assign external users if I don’t "know" them in advance? I’d like other users, on other organizations, to be able to login to our platform using the users’ Okta credentials

**Comment by iainmcgin** [comment]
Sorry that we were similarly non-responsive here, but we don't have an active maintainer for this project anymore (see #444). For provider-specific issues, there's not much help we can provide as we don't have easy access to test environments for each provider.
